setup.cfg

```diff
@@ -38,9 +38,13 @@ keywords =
 packages = find:
 install_requires =
     attrs>=19.2
-    deprecated
     hepunits>=2.0.0
+    check-manifest;extra == dev" or extra == "all"
     importlib-resources>=2.0;python_version<"3.9"
+    pytest>=6;extra == "test" or extra == dev" or extra == "all"
```

Comment:
Maybe it is missing a `"` before `dev`? (In all items that seem to be the case)

Reply:
(Although, it might still not work, because the way setuptools process this is... [complicated](https://github.com/pypa/setuptools/blob/19bfd4d13755011a027013a63a637f02bda0c0cd/setuptools/dist.py#L528)...)

Reply:
I think that was it, seems to be passing now!